public/data/yashsharma999.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Yash Sharma",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/yashsharma.png",
```

I believe it should be `yashsharma999.png`. Please update it with the correct one! 

```suggestion
  "avatar": "https://github.com/yashsharma999.png",
```